packages/cli/src/oclif/commands/invoke.js

```diff
@@ -259,7 +259,7 @@ const getAuthLabel = async (labelTemplate, authData, meta, zcacheTestObj) => {
   const testResult = await testAuth(authData, meta, zcacheTestObj);
   labelTemplate = labelTemplate.replace('__', '.');
   const tpl = _.template(labelTemplate, { interpolate: /{{([\s\S]+?)}}/g });
-  return tpl(testResult);
+  return tpl({ bundle: { authData, inputData: testResult } });
```

Comment:
For `connectionLabel`, these both templates actually work and are equivalent:

- `{{username}}`
- `{{bundle.inputData.username}}`

This PR fixes the second case, but will break the first case. @standielpls can you fix it so that it works for both cases? Here's the corresponding [backend code](https://gitlab.com/zapier/zapier/-/blob/3c9b81f79824f8ca2869cf2957e5331e304ec63b/web/backend/zapier/app_platform/engine/models/authentication.py#L456-457) we should match.